code/datums/components/mind_linker.dm

```diff
@@ -184,7 +184,7 @@
 	return ..()
 
 /datum/component/mind_linker/active_linking/link_mob(mob/living/to_link)
-	if(HAS_TRAIT(to_link, TRAIT_MINDSHIELD)) // Mindshield implant - no dice
+	if(HAS_MIND_TRAIT(to_link, TRAIT_UNCONVERTABLE)) // Mindshield implant - no dice
```

Comment:
the comment seems to imply that this is specifically for mindshield implants... have it check for both unconvertable and mindshield.